### ISSUE TITLE
Examples not training well

### ISSUE THREAD

**Issue by pyeguy** [connorcoley/conv_qsar_fast#3]
I'm running python 3.6 and having some weird output from training the examples... 
It seems like the lib was written in python 2.x but after wrapping a few `range` calls in `list` it seems like things are working (executing) fine except that the example input.cfg's are stopping very early (~12-20 epochs for the Ab-oct example) and generating some pretty horrendous models:
![06-30-2018_01-31 train](https://user-images.githubusercontent.com/5299472/42120151-4e3abe96-7bcb-11e8-9282-799d49e4ab10.png)

Any thoughts on where to start trouble shooting?

Also what were the versions of Rdkit, Theano, and python this was written in?

**Comment by connorcoley** [comment]
That is a pretty atrocious model. If you're using the input .cfg files as they are, then I'm not sure why you would be seeing that. When I've seen cases like this before, it has been because the learning rate hyperparameter is way too high and the model collapses to local minimum where it just predicts the mean value. 

These were written  for Python 2.7.6. I believe the RDKit version was 2013.09.1, the Theano version was 0.9.0dev2.dev-58e93f9b94113d39fab7da6aefaf354cddead2e1, and Keras 1.1.0.

I don't know how more recent versions would affect the models besides causing syntax errors (which it sounds like you've fixed). 

Do you mind trying a couple different hyperparameter settings for the Ab-oct model and seeing if they do any better?
